File: src/average.swift
// データを配列に用意します
var a = [1,3,10,2,8]

// 合計値を0に初期化します
var sum = 0
// データの個数だけくり返します
for i in 0..<a.count {
	// 各値を足します
	sum += a[i]
}
// 平均値を計算します
var average = Float(sum) / Float(a.count)
// 結果を表示します
print("平均=",average)

File: src/cpp/average.cs
using System;

namespace test
{
	internal class Program
	{
		static void Main(string[] args)
		{
			// データを配列に用意します
			int[] a = new int[] {1,3,10,2,8};
			
			// 合計値を0に初期化します
			int sum = 0;
			// 要素の個数を調べます
			int length = a.Length; 
			// データの個数だけくり返します
			int i;
			for (i=0; i<length; i++) {
				// 各値を足します
				sum += a[i];
			}
			// 平均値を計算します
			float average = (float)sum / length;
			
			// 値を表示します
			Console.WriteLine("平均={0} ",average);
		}
	}
}
